app/graphql/mutations/delete_task.rb

```diff
@@ -0,0 +1,15 @@
+module Mutations
+    class DeleteTask < BaseMutation
+      # arguments passed to the `resolve` method
+      argument :id, Integer, required: true
+  
+      # return type from the mutation
+      type Types::TaskType
+  
+      def resolve(id:)
+        task_to_delete = Task.find(id)
+        task_to_delete.destroy
+        task_to_delete
```

Comment:
how are we returning a task_to_delete that is already destroyed?

Reply:
I have seen that some people return the deleted item on their apis.